docs/core/whats-new/index.md

```diff
@@ -1,4 +1,4 @@
 ---
-redirect_url: dotnet-core-3-0
+redirect_url: dotnet-core-3-1
```

Comment:
this page doesn't exist yet. Should you bundle this change and the other one with when the what's new is created?

Reply:
Sorry git didn't pull in the file for some reason. I've redid the push and it's here now.